core/src/main/java/edu/wpi/grip/core/operations/composite/DistanceTransformOperation.java

```diff
@@ -0,0 +1,124 @@
+
+package edu.wpi.grip.core.operations.composite;
+
+import com.google.common.eventbus.EventBus;
+
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import edu.wpi.grip.core.SocketHints;
+
+import java.io.InputStream;
+import java.util.Optional;
+
+import org.bytedeco.javacpp.opencv_core.Mat;
+import static org.bytedeco.javacpp.opencv_core.*;
+import static org.bytedeco.javacpp.opencv_imgproc.*;
+
+/**
+ * GRIP {@link Operation} for
+ * {@link org.bytedeco.javacpp.opencv_imgproc#distanceTransform}.
+ *
+ * @author Sam Carlberg
+ */
+public class DistanceTransformOperation implements Operation {
```

Comment:
What exactly does this operation do?  I can't seem to get it to output anything meaningful